### DISCUSSION TITLE
Can I create multiple users for Vercel Postgre?

### DISCUSSION THREAD

**Discussion by jpvalery** [vercel/community#3521] [Feature Question]
### Question

Hey there,

I tried searching for an answer but couldn't find one. 

Is there a way to create multiple users for the Vercel Postgre integration?

I find it very convenient to have everything in one place, but working on a sync with another system and I'd like to have a read-only user. 

Cheers,

**Comment by amyegan** [comment]
Hey there. Configuring multiple Postgres access roles is not supported at this time. Vercel Postgres is still in [Beta](https://vercel.com/docs/concepts/release-phases#beta) on [Hobby and Pro plans](https://vercel.com/docs/accounts/plans/pro), and you can learn more about current limits at https://vercel.com/docs/storage/vercel-postgres/limits#

**Reply by jpvalery** [reply]
Thanks for the reply, Amy.

Is this being considered for the future or not?
